topcoffea/modules/event_selection.py

```diff
@@ -9,7 +9,10 @@
 def passes_trg_inlst(events,trg_name_lst):
     tpass = np.zeros_like(np.array(events.MET.pt), dtype=bool)
     trg_info_dict = events.HLT
-
+    #print("\n\n\n\n\n\n")
```

Comment:
Is this needed? It looks like maybe a leftover debugging statement that could be removed.

Reply:
That's true, removed it